config/samples/resources/alloydbinstance/read-instance/alloydb_v1beta1_alloydbinstance.yaml

```diff
@@ -31,7 +31,6 @@ spec:
     name: alloydbinstance-dep-read
   instanceTypeRef:
     external: READ_POOL
-  availabilityType: REGIONAL
```

Comment:
AGENT: Correct. Removing `availabilityType` here is appropriate as it is not a valid or required configuration for Read Pool instances, preventing user confusion.